### PR TITLE
Update for Snowbridge SLULA change

### PR DESCRIPTION
**Pull request by colmsnowplow**

### CODE REVIEW

File: docs/contributing/community-license-faq/index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ For purposes of this Agreement, “Excluded Purpose” means making available an
 
 </details>
 
-For example, this does not allow the hosting of Snowplow’s Snowbridge product, or other software licensed under the Snowplow Community License, as part of an online service or distribution of on-premises offerings, that compete with Snowplow products or services.
+For example, this does not allow the hosting of any Community Licence version of Snowplow’s Snowbridge product, or other software licensed under the Snowplow Community License, as part of an online service or distribution of on-premises offerings, that compete with Snowplow products or services.
```
Comment (stanch) [inline docs/contributing/community-license-faq/index.md:24]:
```suggestion
For example, this does not allow the hosting of any Community License version of Snowplow’s Snowbridge product, or other software licensed under the Snowplow Community License, as part of an online service or distribution of on-premises offerings, that compete with Snowplow products or services.
```

**Review by stanch — CHANGES_REQUESTED** [review]

**Comment by stanch** [comment]
Could you add “Snowbridge (2.4.0+)” to this page? https://docs.snowplow.io/docs/contributing/copyright-license/

**Comment by colmsnowplow** [comment]
Done. We should wait to merge until release ofc

File: docs/contributing/copyright-license/index.md [hunk 2]
```diff
@@ -77,6 +77,7 @@ All pipeline apps:
 * [BigQuery Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/index.md) _(2.0.0+)_
 * [Lake Loader](/docs/pipeline-components-and-applications/loaders-storage-targets/lake-loader/index.md) _(0.2.0+)_
 * Other loaders
+* [Snowbridge](/docs/destinations/forwarding-events/snowbridge/) _(2.4.0+)_
```
Comment (stanch) [inline docs/contributing/copyright-license/index.md:80]:
```suggestion
* [Snowbridge](/docs/destinations/forwarding-events/snowbridge/index.md) _(2.4.0+)_
```

**Review by stanch — APPROVED** [review]

**Comment by pondzix** [comment]
Closing in favour of https://github.com/snowplow/documentation/pull/904

**Pull request closed without merging**